connector-runtime/connector-runtime-core/src/main/java/io/camunda/connector/runtime/core/inbound/InboundConnectorContextImpl.java

```diff
@@ -46,6 +49,8 @@ public class InboundConnectorContextImpl extends AbstractConnectorContext
 
   private Health health = Health.unknown();
 
+  private EvictingQueue<Activity> logs = EvictingQueue.create(10);
```

Comment:
We could make the queue size configurable via a property and allow the client to decide how many log entries to request, to avoid changing the code when we decide to show more or less records than 10.